### ISSUE TITLE
If the most recent version is not approved (ex. Mandatory amendment not approved) there is no way for staff to start a new mandatory amendment 

### ISSUE THREAD

**Issue by LisaMoore1** [bcgov/range-web#822]
Amendments can only be created from the most recent approved version. If you had an approved version and then an amendment is submitted and not approved the most recent version in the list is not the approved (amendable) version.

Using three dot version list and selecting the approved version doesn’t take you to a view you can select “Amend” from
Using the WQ lets you see there was an approved version but not get to

**Comment by LisaMoore1** [comment]
@micheal-w-wells I’m not sure if this one is a blocker to the first release on amendments.  I’d say not if we could get this one out quickly without getting hung up in other things. I’ve put in Sprint Backlog even tho I try not to put things there that aren’t hugely essential unless you’ve said that’s where it should go but I want to make sure it doesn’t get lost. Feel free to move it somewhere appropriate.  There will be another issue related to a not-approved mandatory amendment and AHs seeing options they shouldn’t coming shortly.

**Comment by LisaMoore1** [comment]
Working the minor amendment flow I found this...and I have some question about it as well.  Not sure if it should be a separate issue or not.  The “legal version” of this button I think means if I started an amendment I’d be doing it on the current legal (which is right) but it feels wrong to have this option if what I see on the screen is a non-legal version. 

![IMG_8048.jpeg](https://images.zenhubusercontent.com/5d937161c8e4d80001227881/f90811b5-8db3-4e1c-b409-63791ec7c2f8)
